components/blue-print/ui-section.tsx

```diff
@@ -41,8 +42,16 @@ function UISection(section: uiTypes.UISectionObj) {
           <UIData {...obj} key={`UI-Data-${rowIndex}`} />
         )
       )}
+      
     </div>
   )
 }
 
 export default UISection
+
+/*        ) : obj.type === 'world-map' ? (
+          <UIWorldMap {...obj} key={`UI-world-map-${rowIndex}`} />
+        ) : obj.type === 'region-map' ? (
+          <UIRegionMap {...obj} key={`UI-region-map-${rowIndex}`} />
+        ) : obj.type === 'tactical-map' ? (
+          <UITacticalMap {...obj} key={`UI-tactical-map-${rowIndex}`} /> */
```

Comment:
Delete unused code. Don't be afraid to let go. We can add this back in if we need it later.

Reply:
going to be deleted